### ISSUE TITLE
git hub clone points origin wrong

### ISSUE THREAD

**Issue by mattiasb** [seveas/git-spindle#102]
When I run `git hub clone magnars/.emacs.d` I get output like this:

```
Cloning into '.emacs.d'...
[...]
Pointing origin to git@github.com:mattiasb/.emacs.d.git
[...]
```

When I then run `git remote -v show` I see that git-spindle indeed have pointed origin to my repo instead of the repo I cloned. I'm guessing this is because I have a repo with the same name because it doesn't happen when I clone `git-spindle` for example. 

Also, the different `.emacs.d` repos aren't forks or even based off of each other but totally separate Emacs configurations.

**Comment by seveas** [comment]
Intriguing. I can reproduce that here. Looks like it clones the right thing, but then set-origin gets confused.

**Comment by seveas** [comment]
Solution:
- set_origin needs to be less dwim-y (don't try to find a repo in your account)
- fork, clone etc. need to pass a repo to set_origin so it doesn't need the dwimness

Will work on this asap, have another feature to finish first.

**Comment by seveas** [comment]
Should be fixed in 07b1fea
